coral/src/app/features/components/TopicDetailsModalContent.tsx

```diff
@@ -25,9 +25,7 @@ const formatAdvancedConfig = (
     entries.map((config) => [config.configKey, config.configValue])
   );
 
-  const flatObjectToJsonString = JSON.stringify(entriesToFlatObject, null, 2);
-
-  return flatObjectToJsonString;
+  return JSON.stringify(entriesToFlatObject, null, 2);
```

Comment:
I feel like assigning this value to a variable with a clear name served some legibility purpose ("this is now a `JSON` string"). I won't block the PR for this though ^^

Reply:
That's a good point, will change it back.